crud/users.py

```diff
@@ -202,3 +202,66 @@ async def set_new_role(uid, new_role, user_role, db: AsyncSession):
     await db.commit()
     return {'status': 'success',
             'message': f'User {uid} successfully changed to {new_role.role}.'}
+
+
+@handle_errors
+async def set_reminder(user_id, reminder_set, db: AsyncSession):
+    """
+    Function to set the reminders on or off for the user.
+
+    Returns:
+        Message of the transaction.
+    """
+    query = sa.select(User).where(User.id == user_id)
+    result = await db.execute(query)
+    user = result.scalar()
+    if user.daily_reminder == reminder_set and reminder_set is True:
+        raise HTTPException(status_code=400,
+                            detail='Reminders are already enabled for this user')
+    if user.daily_reminder == reminder_set and reminder_set is False:
+        raise HTTPException(status_code=400,
+                            detail='Reminders are already disabled for this user')
+
+    reminders_config = (
+        sa.update(User).where(User.id == user_id).values(daily_reminder=reminder_set)
+    )
+    await db.execute(reminders_config)
+    await db.commit()
+    return {'status': 'success', 'message': 'Reminders successfully configured.'}
+
+
+@handle_errors
+async def send_reminders(timezone, user_role, db: AsyncSession):
+    """
+    Function to send the reminders to users with this function
+
+    Returns:
+        Message of the transaction.
+    """
+    if user_role != 'admin':
+        raise HTTPException(status_code=403,
+                            detail=NOT_AUTHORIZED)
+    tasks_url = app_url + ":8080/api/v1/tasks/"
+    query = sa.select(User.email, func.string_agg(
```

Comment:
I advice you to look at the jinja2 templating libary to construct a new messages.
Also maybe it`ll be better to move rendering message logic to separate function if it`ll be possible